### PR TITLE
Adding functionality for listing file uploads.

### PR DESCRIPTION
**Pull request by wangjohn**
This should allow you to list file uploads via the go bindings.

r? @cosn

### CODE REVIEW

File: fileupload/client_test.go [hunk 1]
```diff
@@ -55,3 +55,40 @@ func TestFileUploadNewThenGet(t *testing.T) {
 		t.Errorf("(GET) File upload id %q does not match expected id %q\n", target.ID, res.ID)
 	}
 }
+
+func TestFileUploadList(t *testing.T) {
+	f, err := os.Open("test_data.pdf")
```
Comment (cosn) [inline fileupload/client_test.go]: is this going to be available locally? Is it checked in already?

**Comment by cosn** [comment]
:+1: 

we should also bump the version in `stripe.go`, and add a changelog entry. I can take care of merging and tagging

File: fileupload/client_test.go (hunk 1 above)
Comment (wangjohn) [inline fileupload/client_test.go]: Yep, it was added for tests for file uploads when they were originally added to the Go bindings (here: https://github.com/stripe/stripe-go/blob/master/fileupload/test_data.pdf)

**Comment by wangjohn** [comment]
Cool, thanks! I've updated the version and changelog.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,6 @@
+4.2.2 2015-01-13
```
Comment (cosn) [inline CHANGELOG]: 4.2.0

File: stripe.go [hunk 3]
```diff
@@ -20,10 +20,10 @@ const (
 )
 
 // apiversion is the currently supported API version
-const apiversion = "2014-12-17"
+const apiversion = "2015-01-11"
```
Comment (cosn) [inline stripe.go]: any other changes between these two versions?

File: stripe.go [hunk 4]
```diff
@@ -20,10 +20,10 @@ const (
...
 
 // clientversion is the binding version
-const clientversion = "4.2.1"
+const clientversion = "4.2.2"
```
Comment (cosn) [inline stripe.go]: 4.3.0

File: stripe.go (hunk 3 above)
Comment (wangjohn) [inline stripe.go]: Looks like these are the only changes made to the go bindings since the last release.

**Pull request merged**